_posts/guides/2019-08-27-guide_the-pbr-shader.md

```diff
@@ -9,31 +9,35 @@ toc_sticky: true
 ---
 <img src="/assets/images/guide_headers/guide_pbr_shader.jpg" alt="PBR Guide" style="display: block; margin: auto;">
 
-The PBR Shader is a handy new tool added to Momentum which allows you to create better looking models with less textures and VMT editing.  
+The PBR shader is a powerful new shader that provides more realistic and versatile rendering of brushes and models within Momentum Mod whilst using less textures and needing fewer VMT tweaks.  
```

Comment:
Would be good to use the un-abbreviated name on first use. I think VMT is self-explanatory though as knowing it is a pre-req.

```suggestion
The Physically Based Rendering (PBR) shader is a powerful new shader that provides more realistic and versatile rendering of brushes and models within Momentum Mod whilst using less textures and needing fewer VMT tweaks.  
```